__tests__/app.test.js

```diff
@@ -15,3 +18,30 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects each with a slug and description property", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        topics.forEach((topic) => {
```

Comment:
This test would pass by default if the array received had a length of zero, how can we test that the endpoint is returning all of the articles?